test/json/test-core.json

```diff
@@ -0,0 +1,140 @@
+[
+  { "src": "hello", "exp": "hello" },
+  { "src": "hello {world}", "exp": "hello world" },
+  {
+    "src": "hello { world\t\n}",
+    "exp": "hello world",
+    "cleanSrc": "hello {world}"
+  },
+  {
+    "src": "hello {\u3000world\r}",
+    "exp": "hello world",
+    "cleanSrc": "hello {world}"
+  },
+  { "src": "hello {|world|}", "exp": "hello world" },
+  { "src": "hello {||}", "exp": "hello " },
+  {
+    "src": "hello {$place}",
+    "params": { "place": "world" },
+    "exp": "hello world"
+  },
+  {
+    "src": "hello {$place-.}",
+    "params": { "place-.": "world" },
+    "exp": "hello world"
+  },
+  {
+    "src": "hello {$place}",
+    "errors": [{ "type": "unresolved-var" }],
+    "exp": "hello {$place}"
+  },
+  {
+    "src": "{$one} and {$two}",
+    "params": { "one": 1.3, "two": 4.2 },
+    "exp": "1.3 and 4.2"
+  },
+  {
+    "src": "{$one} et {$two}",
+    "locale": "fr",
+    "params": { "one": 1.3, "two": 4.2 },
+    "exp": "1,3 et 4,2"
+  },
```

Comment:
These test something other than the `:string` formatter. It might be a good idea to move them with others of the same stripe or segregate them.